src/components/code-block/index.js

```diff
@@ -135,7 +135,14 @@ const CodeBlock = props => {
 		)[1];
 		const firstChild = getChild(child.props);
 		const code = String(firstChild || '').replace(/(^\s+|\s+$)/g, '');
-		return <HighlightedCodeBlock {...props} code={code} lang={lang} />;
+		return (
+			<HighlightedCodeBlock
+				{...props}
+				code={code}
+				lang={lang}
+				key={lang + '\n' + code}
```

Comment:
Same ID we use for the highlight cache -- without the forced remount, we'll have a stale highlight value from the previous page's code block for a render (thereby throwing our positioning off)